### PR TITLE
zcap-ld `Authenticator`s

### PR DESCRIPTION
**Pull request by chunningham**
Implements ZCAP-LD based `Authenticator`s for Tezos and Ethereum, corrosponding to [Kepler PR #49](https://github.com/spruceid/kepler/pull/49) and loosely depending on `didkit-wasm` for the ZCAP implementation.

### CODE REVIEW

**Comment by bumblefudge** [comment]
Discussed 9/2.  190 has been merged to main, but how to refactor/rebase the remaining changes is waiting on another change to port handling, so will remain open until those changes have also been made.

**Review by sbihel — COMMENTED** [review]
I think the `dist/` can be removed now that it's published

File: src/index.test.ts [hunk 1]
```diff
@@ -1,71 +1,122 @@
...
+        // register orbit with allowlist
+        oid = await fetch(`${allowlist}/${params}`, {
+            method: 'PUT',
+            body: JSON.stringify([controller.id()]),
+            headers: {
+                'Content-Type': 'application/json',
+                'X-Hcaptcha-Sitekey': '10000000-ffff-ffff-ffff-000000000001',
+                'X-Hcaptcha-Token': '10000000-aaaa-bbbb-cccc-000000000001'
+            }
+        }).then(async res => res.text());
```
Comment (grsouza) [inline src/index.test.ts:33]: Is it ok for exposing a method for registering the orbit with the allowlist?
Or we don't want to create a dependency between kepler <-> allowlist?

I've used that same code when integrating kepler-sdk, if it's exposed would be better.

Comment (chunningham) [reply src/index.test.ts:33]: yeah IMO it's better to maintain independance between the two. The orbit creation/registration process could in future happen in a really wide variety of ways, which can't be easily enumerated in this package (e.g. email-verification, rebase, fiat or crypto transaction, etc.). The allowlist is itself more of a functional placeholder until those methods are further developed.

**Review by grsouza — APPROVED** [review]
Awesome

**Pull request merged**